.vale/styles/CheDocs/TechnicalTerms.yml

```diff
@@ -0,0 +1,29 @@
+---
+extends: substitution
+ignorecase: false
+level: error
+message: Use '%s' instead of '%s'
+scope: sentence
+swap:
+  '\sContainer Registry': Red Hat Ecosystem Catalog
+  '\sContainer Catalog': Red Hat Ecosystem Catalog
+  '\sconfig map': ConfigMap
+  '\sconfig maps': ConfigMaps
```

Comment:
This is also according to k8s docs style but not OpenShift. OpenShift docs do not pluralize API objects https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references

I'm mostly bringing this up so we as a team decide clearly on the style we want to follow and enforce in the docs.